pages/lessons/index.js

```diff
@@ -40,6 +44,32 @@ const MainWrapper = styled.main`
   margin: 10vw;
 `;
 
+const StyledH2 = styled.h2`
+  position: absolute;
+  bottom: 0;
+  z-index: 100;
+  background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(255, 255, 255, 0.3));
+  color: #ffffff;
+  width: 100%;
+  padding: 0.4em;
+
+  &:hover {
+    background-color: red;
+    cursor: pointer;
+  }
+
+  &:active {
+    background-color: black;
```

Comment:
Never use color values like `black`. These may be interpreted differently by browsers. Use `HEX`, `RGB` or `HSL`